pygame/classify_capture.py

```diff
@@ -14,16 +14,61 @@
 
 """A demo to classify Pygame camera stream."""
 import argparse
-import os
-import io
-import time
+import collections
 from collections import deque
+import io
 import numpy as np
+import operator
+import os
 import pygame
 import pygame.camera
 from pygame.locals import *
+import tflite_runtime.interpreter as tflite
+import time
+
+Class = collections.namedtuple('Class', ['id', 'score'])
```

Comment:
nit: how about "Category"?  Or "Classification" ?

Reply:
Agree "Class" has overloaded meanings in programming languages, and "class" is a reserved keyword.
Unfortunately our official tflite examples also use "classes", "Class" etc. which is unfortunate. 
I agree lets go to "category" or "classification"